presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java

```diff
@@ -693,22 +705,22 @@ public int getId()
 
         public String getPrecision()
         {
-            return precision;
+            return precision.name();
```

Comment:
The getPrecision could return the `precision` and let the (only) caller apply `name()` on their own.